internal/config/config.go

```diff
@@ -33,26 +35,26 @@ type dbConfig struct {
 }
 
 type svcConfig struct {
-	Address               string        `json:"address,omitempty"`
-	AgentEndpointAddress  string        `json:"agentEndpointAddress,omitempty"`
-	CertStore             string        `json:"cert,omitempty"`
-	BaseUrl               string        `json:"baseUrl,omitempty"`
-	BaseAgentEndpointUrl  string        `json:"baseAgentEndpointUrl,omitempty"`
-	BaseUIUrl             string        `json:"baseUIUrl,omitempty"`
-	CaCertFile            string        `json:"caCertFile,omitempty"`
-	CaKeyFile             string        `json:"caKeyFile,omitempty"`
-	SrvCertFile           string        `json:"srvCertFile,omitempty"`
-	SrvKeyFile            string        `json:"srvKeyFile,omitempty"`
-	AltNames              []string      `json:"altNames,omitempty"`
-	LogLevel              string        `json:"logLevel,omitempty"`
-	HttpReadTimeout       util.Duration `json:"httpReadTimeout,omitempty"`
-	HttpReadHeaderTimeout util.Duration `json:"httpReadHeaderTimeout,omitempty"`
-	HttpWriteTimeout      util.Duration `json:"httpWriteTimeout,omitempty"`
-	HttpIdleTimeout       util.Duration `json:"httpIdleTimeout,omitempty"`
-	HttpMaxNumHeaders     int           `json:"httpMaxNumHeaders,omitempty"`
-	HttpMaxHeaderBytes    int           `json:"httpMaxHeaderBytes,omitempty"`
-	HttpMaxUrlLength      int           `json:"httpMaxUrlLength,omitempty"`
-	HttpMaxRequestSize    int           `json:"httpMaxRequestSize,omitempty"`
+	Address                string        `json:"address,omitempty"`
+	AgentEndpointAddress   string        `json:"agentEndpointAddress,omitempty"`
+	CertStore              string        `json:"cert,omitempty"`
+	BaseUrl                string        `json:"baseUrl,omitempty"`
+	BaseAgentEndpointUrl   string        `json:"baseAgentEndpointUrl,omitempty"`
+	BaseUIUrl              string        `json:"baseUIUrl,omitempty"`
+	SrvCertFile            string        `json:"srvCertFile,omitempty"`
+	SrvKeyFile             string        `json:"srvKeyFile,omitempty"`
+	ServerCertName         string        `json:"ServerCertName,omitempty"`
+	ServerCertValidityDays int           `json:"ServerCertValidityDays,omitempty"`
```

Comment:
We should consistently use camelCase in user-facing APIs (actually, we're already mixing with kebab-case, but that's harder to fix now) and spell out.
```suggestion
	SrvCertFile            string        `json:"serverCertificateFile,omitempty"`
	SrvKeyFile             string        `json:"serverKeyFile,omitempty"`
	ServerCertName         string        `json:"serverCertificateName,omitempty"`
	ServerCertValidityDays int           `json:"serverCertificateValidityDays,omitempty"`
```

Reply:
Ack